skademlia/protocol.go

```diff
@@ -207,7 +207,6 @@ func (p Protocol) Server(info noise.Info, conn net.Conn) (net.Conn, error) {
 	go func() {
 		if _, err = p.client.Dial(id.address, WithTimeout(3*time.Second)); err != nil {
 			p.client.logger.Printf("Client %s was not able to be dialed back, closing connection", id)
```

Comment:
We should update this message too
```suggestion
			p.client.logger.Printf("Client %s was not able to be dialed back, not adding as a routable peer", id)
```